packages/frontend/pages/items/filter.tsx

```diff
@@ -16,10 +19,16 @@ export const getServerSideProps = wrapper.getServerSideProps(
 const FilteredPage = () => {
   const router = useRouter();
   const filter = router.query.filter as string;
+  const { list } = useTypedSelector((store) => store.product);
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    dispatch(fetchProducts({ limit: 5, type: ProductType.AUCTION }));
+  }, [dispatch]);
```

Comment:
1. dispatch an action when the filter changes.
2. Filter is not just category - it's an object { limit, category, ... }
3. Move it to the Filter component.

===== Flow =====
All filters will be present in query (but not limit for now). On query changes, please dispatch the action with your object.